docs/geometry.rst

```diff
@@ -1,78 +1,86 @@
 ~~~~~~~~~~~~
 
-For convenience, the ``Geometry`` class can be constructed with a WKT (Well-Known Text)
-or WKB (Well-Known Binary) representation of a geometry:
+For convenience, geometries can be constructed from a WKT (Well-Known Text)
+or WKB (Well-Known Binary) representation:
 
 .. code:: python
 
-  >>> from pygeos import Geometry
-  >>> point_1 = Geometry("POINT (5.2 52.1)")
-  >>> point_2 = Geometry(b"\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\xf0?")
+  >>> from shapely import from_wkb, from_wkt
+  >>> from_wkt("POINT (5.2 52.1)")
+  <shapely.Point POINT (5.2 52.1)>
+  >>> from_wkb(b"\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\xf0?")
+  <shapely.Point POINT (1 1)>
 
 A more efficient way of constructing geometries is by making use of the (vectorized)
-functions described in :mod:`pygeos.creation`.
+functions described in :mod:`shapely.creation`.
 
 Pickling
 ~~~~~~~~
 
 Geometries can be serialized using pickle:
 
   >>> import pickle
-  >>> pickled = pickle.dumps(point_1)
-  >>> pickle.loads(point_1)
-  <pygeos.Geometry POINT (5.2 52.1)>
+  >>> from shapely import Point
+  >>> pickled = pickle.dumps(Point(1, 1))
+  >>> pickle.loads(pickled)
+  <shapely.Point POINT (1 1)>
 
 .. warning:: Pickling will convert linearrings to linestrings.
-             See :func:`pygeos.io.to_wkb` for a complete list of limitations.
+             See :func:`shapely.io.to_wkb` for a complete list of limitations.
 
 Hashing
 ~~~~~~~
 
 Geometries can be used as elements in sets or as keys in dictionaries.
 Python uses a technique called *hashing* for lookups in these datastructures.
-PyGEOS generates this hash from the WKB representation.
+Shapely generates this hash from the WKB representation.
 Therefore, geometries are equal if and only if their WKB representations are equal.
 
 .. code:: python
 
-  >>> point_3 = Geometry("POINT (5.2 52.1)")
+  >>> from shapely import Point
+  >>> point_1 = Point(5.2, 52.1)
+  >>> point_2 = Point(1, 1)
+  >>> point_3 = Point(5.2, 52.1)
   >>> {point_1, point_2, point_3}
-  {<pygeos.Geometry POINT (5.2 52.1)>, <pygeos.Geometry POINT (1 1)>}
+  {<shapely.Point POINT (1 1)>, <shapely.Point POINT (5.2 52.1)>}
 
 .. warning:: Due to limitations of WKB, linearrings will equal linestrings if they contain the exact same points.
-             See :func:`pygeos.io.to_wkb`.
+             See :func:`shapely.io.to_wkb`.
 
 Comparing two geometries directly is also supported.
-This is the same as using :func:`pygeos.predicates.equals_exact` with a ``tolerance`` value of zero.
+This is the same as using :func:`shapely.predicates.equals_exact` with a ``tolerance`` value of zero.
 
   >>> point_1 == point_2
   False
+  >>> point_1 == point_3
+  True
   >>> point_1 != point_2
   True
 
 
-Properties
-~~~~~~~~~~
-
-Geometry objects have neither properties nor methods.
-Instead, use the functions listed below to obtain information about geometry objects.
+Geometry types
```

Comment:
This is intended to be a placeholder for now.  We should probably refactor so that each geometry type gets an individual page